common/frame.go

```diff
@@ -147,6 +148,18 @@ func (f *Frame) inflightRequestsLen() int {
 	return len(f.inflightRequests)
 }
 
+func (f *Frame) getInflightRequest() map[network.RequestID]bool {
```

Comment:
If I understand correctly, this method allows us to clone inflight requests safely. What about calling it `cloneInflightRequests` instead? So we can remember in the future why we're doing this 🤔

By the way, the inflight requests can change anytime by the other parts of the system (IIRC). So, IMO, should we better check if the request is still valid before logging it here:

https://github.com/grafana/xk6-browser/blob/9047580e50c5bba56ef3c6d0ce268ea7dfd8f6ea/common/frame_manager.go#L441

Reply:
Yep, good suggestions.

Resolved in: 544c4f17dca3e95976d30f5adb6fe0befb86db80 & f83aa3bad979c5ead203b19c50bba39ecdf76fbb